src/main/java/org/apache/commons/daemon/support/DaemonConfiguration.java

```diff
@@ -62,6 +62,11 @@
     private final Properties configurationProperties;
     private final Properties systemProperties;
 
+    /**
+     * An empty immutable {@code String} array.
+     */
```

Comment:
This does not need to be public. You could define ONE constant to be package private and reuse it.

Reply:
Changed